css/styles.css

```diff
@@ -92,3 +93,129 @@ button:active {
   background: white;
   border-width: 20px 20px 20px 20px;
 }
+
+.sec-works {
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  row-gap: 0;
+  margin-top: 30px;
+  background-color: #fff;
+  grid-template-columns: auto;
+
+  /* padding-top: 70px;
+  padding-bottom: 70px; */
+}
+
+.tit-projects {
+  margin-left: 40px;
+  margin-top: 40px;
+  margin-bottom: 40px;
+}
+
+.buttons-flex {
+  display: flex;
+  column-gap: 10px;
+}
+
+.li-button {
+  background-color: #98dede;
+  color: #1a2236;
+  font-family: Poppins, sans-serif;
+  font-size: 10.33px;
+  font-weight: 300;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+  padding: 5px 10px;
+  border-radius: 5px;
+  margin-top: 20px;
+}
+
+.project {
+  display: flex;
+  align-items: center;
+  margin-left: 0;
+  width: 100%;
+  background-color: #1a2236;
+  padding: 0;
+  padding-top: 0;
+  flex-direction: column;
+  padding-bottom: 0;
+}
+
+.text-content {
+  display: flex;
+  flex-direction: column;
+  margin-top: 0;
+}
+
+.project img {
+  height: auto;
+  width: 100%;
+}
+
+.pr-description {
+  margin-top: 20px;
+  color: #fff;
+}
+
+.pr-description p {
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 100%;
+  margin-top: 20px;
+  margin-left: 20px;
+  margin-right: 36px;
+  text-align: left;
+}
+
+.project-button {
+  display: inline-block;
+  width: 178px;
+  height: 44px;
+  margin-left: 30px;
+  margin-top: 40px;
+  border: 1px solid white;
+  background-color: #000;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  border-radius: 0;
+  color: #fff;
+}
+
+.menu-hamburg {
+  color: #000;
+  width: 1fr;
+  margin-right: 20px;
+}
+
+.about-myself {
```

Comment:
- [ ] May you mean `.about-me`? I couldn't find the class in the HTML file. Please 🙏🏽  apply the requirements for this section:

> Make sure the content of the "About myself" section is aligned with flexbox.